pkgs/api/package.json

```diff
@@ -12,8 +12,8 @@
   },
   "repository": "open-telemetry/opentelemetry-js",
   "scripts": {
-    "build": "npm run lint:fix-quiet && npm run version && tsc --build tsconfig.all.json && npm run package",
-    "clean": "tsc --build --clean tsconfig.all.json",
+    "build": "npm run lint:fix-quiet && npm run version && tsc --build tsconfig.json tsconfig.esm.json tsconfig.esnext.json && npm run package",
```

Comment:
Investigating the build break, as the `api/` package that was merged was not performing any linting so the package.json didn't contain the additional eslint packages. But even adding locally the auto-fix is not working :-(